README.md

```diff
@@ -1,2 +1,23 @@
 # click-apps-for-openwrt
-OpenWrt based click applications
+
+###1) Application to detect motion using motion click
```

Comment:
Mention the click details in the readme. Add link to mikroe website for the motion click. Also add few lines about mikroBus interface or link to that.
Follow the same format for Readme. Overview, Build instructions, Usage etc.